src/pyuvdata/utils/bls.py

```diff
@@ -243,7 +243,8 @@ def parse_ants(uv, ant_str, *, print_toggle=False, x_orientation=None):
     polarizations = []
     ants_data = uv.get_ants()
     ant_pairs_data = uv.get_antpairs()
-    pols_data = uv.get_pols()
+    # don't use uv.get_pols because that can have n's and e's which are unparsable
```

Comment:
I don't use `get_pols` in anything I use, so forgive me if this is a silly question, but is this something we should file an issue for or is this "a feature, not a bug"?